### PR TITLE
Fix my name, add mailmap file

### PR DESCRIPTION
**Pull request by rpavlik**
My name has been upgraded. I fixed the docs accordingly. In a separate commit, I fixed my committer identity with a mailmap file, taking the opportunity to also clean up other duplicate or typo committer IDs.

No functional change.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2024, Rylie Pavlik
+# SPDX-License-Identifier: Unlicense
+
+Alec Delaney <tekktrik@gmail.com>
+Alec Delaney <tekktrik@gmail.com> <89490472+tekktrik@users.noreply.github.com>
+Alec Delaney <tekktrik@gmail.com> <tekktik@gmail.com>
+Carter Nelson <caternuson@gmail.com>
+Dylan Herrada <dylan.herrada@adafruit.com>
+Dylan Herrada <dylan.herrada@adafruit.com> <33632497+dherrada@users.noreply.github.com>
```
Comment (tannewt) [inline .mailmap]: Please map these to Eva as well.

**Review by tannewt — CHANGES_REQUESTED** [review]
Please map Dylan to Eva as well. Thanks!

File: .mailmap (hunk 1 above)
Comment (rpavlik) [reply .mailmap]: Sorry, happy to fix. Didn't realize that, my deepest apologies to Eva. I had assumed they were a couple.

**Comment by rpavlik** [comment]
Thanks for the heads up, I have corrected the other entry you indicated.

**Review by jepler — APPROVED** [review]
Thank you!

**Comment by jepler** [comment]
The failure is nothing to do with your work. Merging despite red CI, but I'll re-investigate why it's failing (I thought we had put in a fix in circuitpython core for this issue).

File: .mailmap (hunk 1 above)
Comment (tannewt) [reply .mailmap]: No problem. I'm not sure she knew that mailmap existed (I didn't.) :-)

**Pull request merged**